### ISSUE TITLE
Allow players to "restart" the game with their not yet retired dungeon

### ISSUE THREAD

**Issue by RighteousSinner** [miki151/keeperrl#1541]
Since the new map system I suggested seemed a bit too complicated, why not develop a sort of "new game+" where the player can "import" their dungeon/adventurer into a new world with new enemies that can have their stats be buffed by the player inputting a set number. Bonus points if modders can set new types of enemies with said system that only spawn after a set number of game "rounds."

**Comment by GameDoungen** [comment]
I agree with this as a possibility. Maybe you can bring your dungeon through different realms with harder and harder enemies and better and better loop. Some major changes would be needed for this to work. My main problem would be that this can kind of go against the spirit of retired dungeons. This could work though.
